tasks.php

```diff
@@ -1563,17 +1588,28 @@ function TaskComments($tid)
     echo "</td></tr></table></form>";
 }
 
-function NotificationMail($tid, $message)
+function NotificationMail($tid, $message, $new_task = false)
 {
     global $code_url, $tasks_url, $pguser;
-    $notify_setting_all = Settings::get_users_with_setting('taskctr_notice', 'all');
-    $notify_setting_this = Settings::get_users_with_setting('taskctr_notice', $tid);
-    $users_to_notify = array_merge($notify_setting_all, $notify_setting_this);
+    if($new_task)
```

Comment:
With this change, it appears that users with taskcenter_notice = all will no longer receive notifications for new tasks. They are supposed to receive notifications for all new or updated tasks.

(Your earlier comment that there's no way to set "all" is not quite correct. Yes, there is no user interface to set it, but it can be set directly in the database by a Squirrel, and the "all" functionality is being used, and must continue to work as it currently does.)

Reply:
Thanks Walt, I'l do that. Do you think it would it be useful to add a user interface to set "all"?

Reply:
I don't think it's useful for most members to watch everything that happens in the task center, so I vote no on a UI to set the 'all' attribute.